src/quacc/atoms/skzcam.py

```diff
@@ -13,12 +13,489 @@
 from monty.io import zopen
 from monty.os.path import zpath
 
+
+class ElementInfo(TypedDict):
+    core: int
+    basis: str
+    ecp: str
+    ri_scf_basis: str
+    ri_cwft_basis: str
+
+
 if TYPE_CHECKING:
+    from ase.atom import Atom
     from numpy.typing import NDArray
 
 has_chemshell = find_spec("chemsh") is not None
 
 
+def create_orca_eint_blocks(
+    embedded_adsorbed_cluster: Atoms,
+    quantum_cluster_indices: list[int],
+    ecp_region_indices: list[int],
+    element_info: ElementInfo | None = None,
+    pal_nprocs_block: dict[str, int] | None = None,
+    method_block: dict[str, str] | None = None,
+    scf_block: dict[str, str] | None = None,
+    ecp_info: dict[str, str] | None = None,
+    include_cp: bool = True,
+    multiplicity: dict | None = None,
+) -> tuple[str, str, str]:
+    """
+    Creates the orcablocks input for the ORCA ASE calculator.
+
+    Parameters
+    ----------
+    embedded_adsorbed_cluster
+        The ASE Atoms object containing the atomic coordinates and atomic charges from the .pun file, as well as the atom type.
```

Comment:
Despite the type hints, a lot of these arguments are not obvious in terms of what should be supplied by the user. 

In this example, it might help to indicate that this Atoms object is typically produced from `[quacc.atoms.skzcam.create_skzcam_clusters][]`, which will autolink to the docs page.

Same comment for `quantum_cluster_indices` and `ecp_region_indices` below.

Almost makes me wonder if `create_skzcam_clusters` should just return a single `dict` with three key/value pairs instead if that three-element tuple is meant to be passed around as a single object. If this is the case, I'd suggest making this change (and updating the `create_skzcam_clusters` type hint to reflect that with a new `TypedDict`).